app/src/main/java/com/framgia/fsalon/data/source/remote/CustomerBookingRemoteDataSource.java

```diff
@@ -0,0 +1,55 @@
+package com.framgia.fsalon.data.source.remote;
+
+import com.framgia.fsalon.data.model.BookingCustomer;
+import com.framgia.fsalon.data.model.BookingCustomerResponse;
+import com.framgia.fsalon.data.source.BookingCustomerDatasource;
+import com.framgia.fsalon.data.source.api.FSalonApi;
+import com.framgia.fsalon.utils.Utils;
+
+import java.util.HashMap;
+import java.util.List;
```

Comment:
ERROR: Unused import: java.util.List. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports